code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -34,6 +34,57 @@
 	if(iscarbon(M))
 		M.emote("scream")
 
+//does low toxin damage, but creates fragile spores when expanding or killed by weak attacks
+/datum/reagent/blob/sporing_pods
+	name = "Sporing Pods"
+	id = "sporing_pods"
+	description = "will do low toxin damage and produce fragile spores when killed or on expanding."
+	color = "#E88D5D"
+	message_living = ", and you feel sick"
+
+/datum/reagent/blob/sporing_pods/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	reac_volume = ..()
+	M.apply_damage(0.4*reac_volume, TOX)
+
+/datum/reagent/blob/sporing_pods/damage_reaction(obj/effect/blob/B, original_health, damage, damage_type, cause)
+	if(!isnull(cause) && damage < 20 && original_health - damage <= 0 && prob(50)) //if the cause isn't fire or a bomb, the damage is less than 20, we're going to die from that damage, 50% chance of a shitty spore.
+		B.visible_message("<span class='warning'><b>A spore floats free of the blob!</b></span>")
+		var/mob/living/simple_animal/hostile/blob/blobspore/weak/BS = new/mob/living/simple_animal/hostile/blob/blobspore/weak(B.loc)
+		BS.overmind = B.overmind
+		BS.update_icons()
+		B.overmind.blob_mobs.Add(BS)
+	return ..()
+
+/datum/reagent/blob/sporing_pods/expand_reaction(obj/effect/blob/B, turf/T)
+	if(prob(10))
+		var/mob/living/simple_animal/hostile/blob/blobspore/weak/BS = new/mob/living/simple_animal/hostile/blob/blobspore/weak(T)
+		BS.overmind = B.overmind
+		BS.update_icons()
+		B.overmind.blob_mobs.Add(BS)
+
+//does brute damage but can replicate when damaged and has a chance of expanding again
```

Comment:
this replicating foam blob looks stupid op
how do you even kill it if by the time you weld it you get 2 new ones